shared/chat/conversation/messages/wrapper/wrapper-author/container.js

```diff
@@ -86,7 +86,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps: OwnProps) => {
   let isErrorFixable = false
   if ((message.type === 'text' || message.type === 'attachment') && message.errorReason) {
     failureDescription = message.errorReason
-    if (stateProps.isYou && message.submitState === 'pending') {
+    if (stateProps.isYou && ['pending', 'failed'].includes(message.submitState)) {
       // This is a message still in the outbox, we can retry/edit to fix
       failureDescription = stateProps.isYou ? `Failed to send: ${message.errorReason}` : message.errorReason
```

Comment:
Why do we check `isYou` again here? Is the first check for it wrong?

Reply:
Hm no, just an oversight. Thanks!